### ISSUE TITLE
NTR, pulmo: Evelvated residual volume (RV)

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6445]
**Preferred term label:** Evelvated residual volume (RV)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sign for Air trapping and Hyperinflation. Comment: Hyperinflated lungs occur when air gets trapped in the lungs and causes them to overinflate. Hyperinflated lungs can be caused by blockages in the air passages or by air sacs that are less elastic, which interferes with the expulsion of air from the lungs

**Parent term (use hpo.jax.org/app)**
Abnormal spirometry test HP:0032340

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
See #6446 
Proposed extended definition

An increased residual volume (RV), which is the amount of air remaining after maximal expiration. RV can be measured by inert gas dilution, nitrogen washout, and (3) body plethysmography. An elevated RV can be interpreted as a sign of air trapping and hyperinflation. PMID:12934788

Can we discuss the proper parent in the ontology? @chrapp, @mgriese, @kknoflach

**Comment by GrieseAG** [comment]
If the differentiation by method is necessary, we can add the parent term abnormal body plethysmography?
Than the abnormality of the volumes will be child terms: 
Evelvated residual volume (RV) #6445; Elevated RV/TLC ratio #6446; Reduced total lung capacity HP:0033169 (?)

and the terms, describing the flow or the volume as a function of time, belongs to Abnormal spirometry test?
We could discuss that tomorrow? 

@katrinknoflach

**Comment by pnrobinson** [comment]
added
